src/js/containers/search/SearchContainer.jsx

```diff
@@ -86,6 +86,13 @@ const SearchContainer = ({ history }) => {
     const [downloadInFlight, setDownloadInFlight] = useState(false);
     const [generateHashInFlight, setGenerateHashInFlight] = useState(false);
     const request = useRef(null);
+    const areAppliedFiltersEmptyRef = useRef();
+
+    useEffect(() => {
+        areAppliedFiltersEmptyRef.current = areAppliedFiltersEmpty;
+    });
```

Comment:
We should only update this if areAppliedFiltersEmpty changes right?

Reply:
https://reactjs.org/docs/hooks-faq.html#how-to-get-the-previous-props-or-state

Reply:
Yes, Thank you.  Question still stands. This `useEffect` runs anytime the component updates, does the `ref` current need to be updated from `false` to `false` or `true` to `true`?